### PR TITLE
Feat: Add payout claimed notification

### PR DESCRIPTION
**Pull request by davecreaser**
Add a new notification for expenditures when a payout is claimed.

CDapp PR here: https://github.com/JoinColony/colonyCDapp/pull/3400

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]
Nice and clean addition :fire: 
Tested as part of its CDapp counterpart and can confirm it works!

**Review by mmioana — APPROVED** [review]
Nice and straight-forward change 💯 👏 

Tested as part of the [colonyCDapp](https://github.com/JoinColony/colonyCDapp/pull/3400#pullrequestreview-2381377584)

File: src/handlers/expenditures/expenditurePayoutClaimed.ts [hunk 1]
```diff
@@ -25,6 +27,7 @@ export default async (event: ContractEvent): Promise<void> => {
     slot,
     token: tokenAddress,
     tokenPayout: amountWithoutFee,
+    agent: initiatorAddress,
```
Comment (jakubcolony) [inline src/handlers/expenditures/expenditurePayoutClaimed.ts:30]: Just checking that you want this notification to be sent to the user who claimed the payout, and not the recipient/owner of the expenditure?

**Review by jakubcolony — APPROVED** [review]
LGTM, tested with colonyCDapp#3400.

File: src/handlers/expenditures/expenditurePayoutClaimed.ts (hunk 1 above)
Comment (davecreaser) [reply src/handlers/expenditures/expenditurePayoutClaimed.ts:30]: It should just be sent colony wide, it's not a specific notification. The initiator address will be used to get information on who triggered the notification (creator), but in reality is isn't really used for anything other than debugging.

**Pull request merged**